README.md

```diff
@@ -103,6 +104,10 @@ Then, run the following command to start the app
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live site](https://budget-it-service.onrender.com/)
+
```

Comment:
- [ ] Good job adding the live demo link to the readme. However, as per the last reviewer's request, kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).